doc_SAI-Proposal-SAI-PTF.md

```diff
@@ -0,0 +1,723 @@
+# SAI PTF
```

Comment:
I think this file should not be uploaded to DASH since it's going to live under SAI. Could you instead make another entry in the table under https://github.com/Azure/DASH/tree/main/test/docs#table-of-contents---dash-test-documentation like you did for the SAI-PTF User guide? Once your proposal is upstreamed we can change the URL to point there. For now it can point to your dev branch. Thanks!

Reply:
@reshmaintel to review

Reply:
Added reference to design instead of the doc itself.